core/Archive.php

```diff
@@ -493,6 +493,9 @@ protected function get($archiveNames, $archiveDataType, $idSubtable = null)
 
         $result = new Archive\DataCollection(
             $dataNames, $archiveDataType, $this->params->getIdSites(), $this->params->getPeriods(), $this->params->getSegment(), $defaultRow = null);
+        if (empty($archiveNames)) {
```

Comment:
should that be dataNames? if not, could maybe exit right in the top? Also what if `archiveNames` is eg `null` or `false` would then this still be detected? or maybe not if it is `[null]`?

Reply:
It could be either. We need to return an empty DataCollection value so the result is properly created (eg, for DataTables we need correct metadata).

I'll add extra code for empty values.